src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -385,6 +389,8 @@ private static String suppressVariationsInInjectedValues(String content) {
     private static String replaceUnpredictableValuesWithPlaceholders(String content) {
         Date now = new Date();
         SimpleDateFormat sdf = new SimpleDateFormat("EEE, dd MMM yyyy, ");
+        //A page can have only one time zone as we cannot have session with two different time zones.
```

Comment:
A page can have two time zones since instructor comments can show comments from different courses and sessions. You may want to correct your comment.